app/views/timeslots/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Timeslots</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Time slot</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @timeslots.each do |timeslot| %>
+      <tr>
+        <td><%= timeslot.start_time %></td>
+        <td><%= link_to 'Show', timeslot %></td>
+        <td><%= link_to 'Edit', edit_timeslot_path(timeslot) unless current_user&.customer? %></td>
```

Comment:
Duplicated check for user role

Reply:
Line 18-19

Reply:
@BohdanChaban  this entry should be, because the button to destroy should not be visible to the customer.